source/getting-started/troubleshooting.markdown

```diff
@@ -0,0 +1,62 @@
+---
+title: "Troubleshooting installation problems"
+description: "Common installation problems and their solutions."
+---
+
+It can happen that you run into trouble while installing and onboarding Home Assistant. This page is here to help you solve the most common problems.
+
+## Can’t access Home Assistant in my browser
+
+### Symptom: “This site can’t be reached”
+
+When trying to access Home Assistant in the browser, the browser shows the message “This site can’t be reached”.
+
+#### Description
+
+This means the browser can’t find your Home Assistant installation on the network.
+
+#### Resolution
+
+To resolve this issue, try the following steps:
```

Comment:
Good practice is going from physical layer up, I'd reorder:

3. -> 4. -> 5. -> 2. -> 1.

Reply:
yes, that would be a very systematic approach. But I think steps 1 and 2 are more likely to be the cause? I was trying to get the most likely causes out of the way first.